src/tests/test.utils.tsx

```diff
@@ -18,17 +22,23 @@ interface IComponentProps {
 }
 
 /**
- *
- *
- *
+ * Will be wrapping all of our tests.
+ * Need to wrap tests in Provider + Persist + Router. Might need to add more later.
+ * This is so rendered components can have access to store.
  */
 
 export const TestWrapper = ({ children }: IComponentProps): JSX.Element => {
     return (
         <Provider store={store}>
```

Comment:
next time we talk, I'll should you some other things you can do here to make your [testing] life a little easier. Being able to setup a `state` and pass it in is the top item on that list.

You'll find, as you write more and more tests, that frequently the hardest part of testing is state building.